### ISSUE TITLE
Mismatch Curlys after updating to latest git

### ISSUE THREAD

**Issue by ecdarreola** [rathena/rathena#2964]
* **rAthena Hash**: 42dda25

* **Client Date**: Won't Matter

* **Server Mode**: Renewal

* **Description of Issue**: Mismatch of Curlys in the Item_db.txt
  * Result: Mismatch Curlys
  * Expected Result: No Mismatch
  * How to Reproduce: Update to latest rAthena
See image below
![image](https://user-images.githubusercontent.com/5693492/37211810-1e2baeda-23e8-11e8-9bec-c0ef53d21d4f.png)

  * Official Information: N/A

Before this commit: https://github.com/rathena/rathena/commit/d7f226a66bcdee7865702451da3e692de2685cd7

It works fine.
  

* **Modifications that may affect results**: Gepard (Not a problem using Windows)

**Comment by aleos89** [comment]
![image](https://user-images.githubusercontent.com/3521753/37212538-3f5c3658-237d-11e8-8b27-59f940556414.png)
```
[Status]: Done reading '11652' entries in 'db/re/item_db.txt'.
[Status]: Done reading '0' entries in 'db/import/item_db.txt'.
```
Seems rAthena master (42dda25) doesn't match up with your output. Also Travis nor AppVeyor are showing any errors.

**Comment by ecdarreola** [comment]
@aleos89 : This is just a cut version of the error. The error was too long that scrolling up cannot be handled by putty. or lagging on my side. I just used my snipping tool and ready it just to get a snippet.

**Comment by ecdarreola** [comment]
@aleos89 : Please see start error

![image](https://user-images.githubusercontent.com/5693492/37213282-8b42b730-23ec-11e8-9c4f-54d33d6b4d27.png)

**Comment by aleos89** [comment]
Did you properly merge this d7f226a?

**Comment by ecdarreola** [comment]
Yes @aleos89 : my itemdb.cpp is an exact replica of that of default rA.

**Comment by Lemongrass3110** [comment]
Which OS and compiler are you using.
Basically neither we nor any of our CI tools have this issue.

**Comment by ecdarreola** [comment]
@Lemongrass3110 : Centos 7 64 bit.

**Comment by ecdarreola** [comment]
@Lemongrass3110 : I tested it in windows platform there's no error in running the server though. All files in the item_db were read correctly but using centos 7 64 bit it has problem. Tested with gepard.

**Comment by laziem** [comment]
I have the same problem. I'm using centos with gepard

**Comment by Daegaladh** [comment]
Well, try with a clean rAthena, if works the problem is obvioulsy Gepard XD

**Comment by ecdarreola** [comment]
Thanks @Daegaladh , please use it on centos 7 64bit. I tried it working with windows. No problem at all with or without gepard. But in centos 7, with gepard it have problem so I assume its not on Gepard.

**Comment by Daegaladh** [comment]
Ok, what's your GCC version?

**Comment by ecdarreola** [comment]
@Daegaladh 7 or devtoolset-7. I just used the installation guide of rA for centos.

**Comment by vstumpf** [comment]
Using gcc 7.2.1 and centos 7 I don't have this issue.
```
[vagrant@localhost rathena]$ gcc --version
gcc (GCC) 7.2.1 20170829 (Red Hat 7.2.1-1)
[vagrant@localhost rathena]$ uname -r
3.10.0-693.17.1.el7.x86_64
[vagrant@localhost rathena]$ cat /etc/*-release
CentOS Linux release 7.4.1708 (Core)
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"

CentOS Linux release 7.4.1708 (Core)
CentOS Linux release 7.4.1708 (Core)
[vagrant@localhost rathena]$ git log --oneline -n 10
9bbb4cb Corrected Lightning Walk requirements (#2984)
3783225 Corrected guild storage saving (#2957)
```

**Comment by Daegaladh** [comment]
Now happens to me on Ubuntu 17.10 with GCC 7.2.0
